Source/WebGPU/WebGPU/Pipeline.mm

```diff
@@ -41,7 +43,7 @@
         if (const auto* pipelineLayoutHint = shaderModule.pipelineLayoutHint(entryPoint)) {
             if (*pipelineLayoutHint == *pipelineLayout) {
                 if (const auto* entryPointInformation = shaderModule.entryPointInformation(entryPoint))
-                    return { { shaderModule.library(), *entryPointInformation } };
+                    return { { shaderModule.library(), *entryPointInformation,  wgslConstantValues } };
```

Comment:
don't we need to populate wgslConstantValues here too?

Reply:
oh we never encounter this path because `ShaderModule::m_pipelineLayoutHints` is always empty, please ignore